### ISSUE TITLE
Some examples give compile error

### ISSUE THREAD

**Issue by jpnielsen** [docb/dbRackCsound#4]
On windows 11, Rack 2.1.2, dbRackCsoiund 2.0.1

Many presets give a blinking red light, when they compile after load.

One such example is preset->fm->fmb3 = blinks red and in log.txt:
```

[1660.744 info src/DBCSound.hpp:88 compile] compile error -1
[1660.744 info src/DBCSound.hpp:89 compile] instr main
ifn ftgenonce 0,0,2^15,10,1
kfreq chnget sprintf("FREQ%d",p4)
kc1 chnget "P1"
kc1 scale2 kc1,0,5,0,10
kc2 chnget "P2"
kc2 scale2 kc2,0,5,0,10
kdepth chnget "P3"
kdepth scale2 kdepth,0,0.2,0,10
ao fmb3 0.7,kfreq, kc1, kc2, kdepth, 6
aenv madsr 0.001,0,1,0.02
ao*=aenv 
outs ao,ao
endin

```
```

I have not yet tried to load all the presets:

├───CsoundModule
│   │   gbuzz.vcvm (RED)
│   │   init.vcvm (GREEN)
│   │   oscil.vcvm (RED)
│   │   perfl.vcvm (RED)
│   │   pluck.vcvm (GREEN)
│   │   repluck.vcvm (RED)
│   │   squine.vcvm (GREEN)
│   │
│   ├───01_vco
│   │       vco2sawtri.vcvm  (GREEN)
│   │       vco2square.vcvm (GREEN)
│   │       vco2userdef.vcvm (GREEN)
│   │       vcoisaw.vcvm (GREEN)
│   │       vcosquareinst.vcvm (GREEN)
│   │       vco_rnd_even.vcvm (GREEN)
│   │       vco_rnd_square.vcvm (GREEN)
│   │
│   ├───02_fm
│   │       fmb3.vcvm (RED)
│   │       fmbell.vcvm (RED)
│   │       fmrhodes.vcvm (RED)
│   │       fmvoice.vcvm (RED)
│   │       foscil.vcvm (RED)
```

**Comment by docb** [comment]
see #3

**Comment by docb** [comment]
fixed in 2.0.2
